task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksList.vue

```diff
@@ -17,6 +18,11 @@
         default: () => [],
       }
     },
+    methods:{
+      updateTaskCompleted(e){
+        window.setTimeout(() => this.tasks = this.tasks.filter((t) => t.task.id !== e.id), 500);
```

Comment:
Should this take in con,sideration wether the show completed filter is activated or not ?

Reply:
it shouldn't take considered when the show completed filter is activated, I ass PR for this, thanks